docs/pvacbind.rst

```diff
@@ -0,0 +1,18 @@
+.. image:: images/pVACbind_logo_trans-bg_sm_v4b.png
+    :align: right
+    :alt: pVACbind logo
+
+pVACbind
+====================================
+
+This component of the pVACtools is used to predict neoantigens for the peptides in a fasta file.
```

Comment:
The copyeditor in me notes that `fasta` was capitalized `FASTA` in the index.rst. 
 Searching the repo, it appears that both versions are used several times.